docs/docs_src/guide/Model.md

```diff
@@ -362,7 +362,7 @@ The other method you can use is by using specific update types. These update typ
 
 - `$SET` - This method will set the attribute to the new value (as shown above)
 - `$ADD` - This method will add the value to the attribute. If the attribute is a number it will add the value to the existing number. If the attribute is a list, it will add the value to the list. Although this method only works for sets in DynamoDB, Dynamoose will automatically update this method to work for lists/arrays as well according to your schema. This update type does not work for any other attribute type.
-- `$REMOVE` - This method will remove the attribute from the item. Since this method doesn't require values you can pass in an array of attribute names.
+- `$REMOVE` - This method will remove the attribute from the item. Since this method doesn't require values, You can pass in an array of attribute names.
```

Comment:
```suggestion
- `$REMOVE` - This method will remove the attribute from the item. Since this method doesn't require values, you can pass in an array of attribute names.
```